test/rule_enforcer.test.ts

```diff
@@ -154,25 +155,25 @@ describe("Filename: rules_enforcer.test.ts:\n\nRules Enforcer", () => {
 		let agents: Agent[] = [red_p_one, red_p_two, red_p_three, blue_p_one, blue_p_two];
 
 		// TODO: Use utility method for getting ops
-		let ops: Operative[] = agents.filter(agent => agent.role === Turn.op) as Operative[];
+		let ops: Teams<Operative> = gu.getOperatives(agents);
 		let select_card = new Card("select");
 		let decoy_card = new Card("decoy");
 
-		expect(re.canSubmitGuess(ops, Team.red)).to.be.false;
+		expect(re.canSubmitGuess(gu.getByTeam(ops, Team.red))).to.be.false;
```

Comment:
I'd generally avoid two letter variables like `re` or `gu`. Can you change those to be `RuleEnforcer` and `Game Utility` instead? That seems much clearer and removes the need for doing the `import ... as`. This should be changed across the board.

Reply:
I tend to agree in general but value the brevity of renaming imports in situations like this (see Lodash imported as _)